sdk/@launchdarkly/observability-python/ldobserve/otel/sampling_log_exporter.py

```diff
@@ -0,0 +1,89 @@
+from typing import List, Optional, Dict, Union, Sequence, Tuple
+from opentelemetry.sdk._logs import LogRecord, LogData  # type: ignore Not actually private.
```

Comment:
`_logs` is experimental, but not really private. The vscode python extension isn't super pleased about it.